code/modules/mob/living/simple_animal/friendly/farm_animals.dm

```diff
@@ -169,9 +169,9 @@
 	speak_chance = 2
 	turns_per_move = 2
 	butcher_results = list(/obj/item/reagent_containers/food/snacks/meat = 1)
-	response_help  = "pets the"
+	response_help  = "pets"
 	response_disarm = "gently pushes aside the"
-	response_harm   = "kicks the"
+	response_harm   = "kicks"
 	attacktext = "kicks"

```

Comment:
I don't even understand why you had to remove the word "pony" at all. Pony is not a trademark/copyrighted thing and doesn't really need to be removed?

Reply:
its not about the word pony, its about it being litteraly MY LITTLE PONY sprites, pony is kept for anyone who cares to sprite a non reference pony

Reply:
So why remove the "kicks the pony" or "pushes aside the pony" or "pets the pony". Those changes were made but not really explained. If you're going to remove sprites you should be replacing the sprite with a different one. Otherwise what even is the point of an unfinished PR?

Reply:
@Pixelzilla "pony" has not been removed, only the subtypes of it, and their icons. The responses are only being edited to prevent instances of `the the`.